cves/CVE-2012-6617.yml

```diff
@@ -146,10 +144,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: There were a lot of unit tests that covered the server of FFmpeg
```

Comment:
Any specific ones worth mentioning, since there are so many?

Reply:
Not really. Mostly cause all of them haven't been updated in a while (The server has been depreciated)